src/main/java/org/example/p21_stringIntoInteger.java

```diff
@@ -0,0 +1,10 @@
+package org.example;
+
+public class p21_stringIntoInteger {
```

Comment:
Use UpperCamelCase formatting for naming classes.

Reply:
Done